packages/cms/src/app/(auth)/tokens/[id]/_actions/index.ts

```diff
@@ -74,16 +76,20 @@ export const updateToken = actionClient
             }
           }),
         )
-        await db.tokenMeta.upsert({
-          tokenId: id,
-          externalId: '',
-          source: 'Overrides',
-          symbol: customMeta?.symbol ?? null,
-          name: customMeta?.name ?? null,
-          logoUrl: customMeta?.logoUrl ?? null,
-          decimals: customMeta?.decimals ?? null,
-          contractName: customMeta?.contractName ?? null,
-        })
+        await db.tokenMeta.deleteByTokenId(id)
```

Comment:
why delete when upserting?

Reply:
Without delete here we're left with e.g. CoinGecko source meta without externalId.